packages/sdks/src/functions/track/index.ts

```diff
@@ -65,6 +65,13 @@ const getTrackingEventData = async ({
   };
 };
 
+export type CustomApiBaseUrl = {
+  /**
+   * Optional override of the `baseUrl` of the Builder API. (Defaults to global `https://cdn.builder.io`)
+   */
+  apiBaseUrl?: string;
```

Comment:
I would rename this to `apiHost` and the description like this:

```suggestion
  /**
   * Sets the host of Builder API calls. (Defaults to global `https://cdn.builder.io`)
   */
  apiHost?: string;
```